includes/embeds/class-amp-twitter-embed-handler.php

```diff
@@ -19,6 +19,7 @@ class AMP_Twitter_Embed_Handler extends AMP_Base_Embed_Handler {
 	/**
 	 * Default width.
 	 *
+	 * @phpstan-ignore-next-line
```

Comment:
This change and the other phpstan-related changes can be removed after rebasing/merging onto `develop` ater the merging of https://github.com/ampproject/amp-wp/pull/6423.